version.properties

```diff
@@ -0,0 +1,6 @@
+version.buildmeta=
+version.major=1
+version.minor=0
+version.patch=1
+version.prerelease=SNAPSHOT
+version.semver=1.0.1-SNAPSHOT
```

Comment:
We should be on 1.0.0-SNAPSHOT right now, there are no previous releases of the linter

Reply:
changed.